data/src/test/java/org/apache/iceberg/data/TestMetricsRowGroupFilterTypes.java

```diff
@@ -222,52 +222,48 @@ public void createParquetInputFile(List<Record> records) throws IOException {
-        new Object[] { "parquet", "time", "10:02:34.000000", "10:02:34.000001" },
-        new Object[] { "parquet", "timestamp",
-            "2018-06-29T10:02:34.000000",
-            "2018-06-29T15:02:34.000000" },
-        new Object[] { "parquet", "timestamptz",
-            "2018-06-29T10:02:34.000000+00:00",
-            "2018-06-29T10:02:34.000000-07:00" },
-        new Object[] { "parquet", "string", "tapir", "monthly" },
-        // new Object[] { "parquet", "uuid", uuid, UUID.randomUUID() }, // not supported yet
-        new Object[] { "parquet", "fixed", "abcd".getBytes(StandardCharsets.UTF_8), new byte[] { 0, 1, 2, 3 } },
-        new Object[] { "parquet", "binary", "xyz".getBytes(StandardCharsets.UTF_8), new byte[] { 0, 1, 2, 3, 4, 5 } },
-        new Object[] { "parquet", "int_decimal", "77.77", "12.34" },
-        new Object[] { "parquet", "long_decimal", "88.88", "12.34" },
-        new Object[] { "parquet", "fixed_decimal", "99.99", "12.34" },
+        { "parquet", "boolean", false, true },
+        { "parquet", "int", 5, 55 },
+        { "parquet", "long", 5_000_000_049L, 5_000L },
+        { "parquet", "float", 1.97f, 2.11f },
+        { "parquet", "double", 2.11d, 1.97d },
+        { "parquet", "date", "2018-06-29", "2018-05-03" },
+        { "parquet", "time", "10:02:34.000000", "10:02:34.000001" },
+        { "parquet", "timestamp", "2018-06-29T10:02:34.000000", "2018-06-29T15:02:34.000000" },
+        { "parquet", "timestamptz", "2018-06-29T10:02:34.000000+00:00", "2018-06-29T10:02:34.000000-07:00" },
+        { "parquet", "string", "tapir", "monthly" },
+        // { "parquet", "uuid", uuid, UUID.randomUUID() }, // not supported yet
+        { "parquet", "fixed", "abcd".getBytes(StandardCharsets.UTF_8), new byte[] { 0, 1, 2, 3 } },
+        { "parquet", "binary", "xyz".getBytes(StandardCharsets.UTF_8), new byte[] { 0, 1, 2, 3, 4, 5 } },
+        { "parquet", "int_decimal", "77.77", "12.34" },
+        { "parquet", "long_decimal", "88.88", "12.34" },
+        { "parquet", "fixed_decimal", "99.99", "12.34" },
 
-        new Object[] { "orc", "boolean", false, true },
-        new Object[] { "orc", "int", 5, 55 },
-        new Object[] { "orc", "long", 5_000_000_049L, 5_000L },
-        new Object[] { "orc", "float", 1.97f, 2.11f },
-        new Object[] { "orc", "double", 2.11d, 1.97d },
-        new Object[] { "orc", "date", "2018-06-29", "2018-05-03" },
-        new Object[] { "orc", "time", "10:02:34.000000", "10:02:34.000001" },
+        { "orc", "boolean", false, true },
+        { "orc", "int", 5, 55 },
+        { "orc", "long", 5_000_000_049L, 5_000L },
+        { "orc", "float", 1.97f, 2.11f },
+        { "orc", "double", 2.11d, 1.97d },
+        { "orc", "date", "2018-06-29", "2018-05-03" },
+        { "orc", "time", "10:02:34.000000", "10:02:34.000001" },
         // Temporarily disable filters on Timestamp columns due to ORC-611
-        // new Object[] { "orc", "timestamp",
+        // { "orc", "timestamp",
```

Comment:
I checked on this, and ORC-611 is closed and the fix versions are 1.6.4 and 1.7.0.

I'm going to see if I can get these tests to pass or open a ticket about possibly upgrading our ORC version if not.

Reply:
I tried updating but it did not affect the tests. There were several orc related files on the compile time path, but I'm not sure what ran during the tests.

It looks like the issue ORC-611 was closed, but it still exists in https://issues.apache.org/jira/browse/HIVE-23036

Reply:
I'm going to also try 1.7.0, but I'm wondering if we aren't fully excluding some ORC dependencies from other dependencies.

Reply:
Turns out 1.7.0 is not published to maven. I am keeping these tests commented out with a mention of the corresponding still open HIVE ticket, as it indicates where in the OrcInputFormat for Hive is the issue. Might be of use to us.